### PR TITLE
CHAD-9229 - hides configuration events

### PR DESCRIPTION
**Pull request by KKlimczukS**
@greens 
I added `{ visibility = { displayed = false } }` to configuration events only.
I didn't change anything in the unit tests yet (to see how the testing framework will handle those changes).
@SmartThingsCommunity/srpol-pe-team

### CODE REVIEW

**Review by greens — COMMENTED** [review]
Yeah. Looks good, but as you can see the tests will need to be updated as visibility is taken into account.

You should be able to run tests locally, though.

**Comment by KKlimczukS** [comment]
@greens @k-morawski 
I'll rebase that branch and add some unit tests here.

File: drivers/SmartThings/zigbee-thermostat/src/init.lua [hunk 1]
```diff
@@ -133,14 +133,14 @@ local thermostat_operating_state_handler = function(driver, device, operating_st
...
-  device:emit_event(ThermostatFanMode.supportedThermostatFanModes(SUPPORTED_FAN_MODES[fan_mode.value]))
+  device:emit_event(ThermostatFanMode.supportedThermostatFanModes(SUPPORTED_FAN_MODES[fan_mode.value], { visibility = { displayed = false } }))
 end
 
 local thermostat_fan_mode_handler = function(driver, device, attr_fan_mode)
   if (FAN_MODE_MAP[attr_fan_mode.value]) then
     local supported_fan_modes = device:get_latest_state("main", ThermostatFanMode.ID, ThermostatFanMode.supportedThermostatFanModes.NAME)
     if supported_fan_modes then
-      device:emit_event(FAN_MODE_MAP[attr_fan_mode.value]({data = {supportedThermostatFanModes = supported_fan_modes}}))
+      device:emit_event(FAN_MODE_MAP[attr_fan_mode.value]({data = {supportedThermostatFanModes = supported_fan_modes, visibility = { displayed = false }}}))
```
Comment (greens) [inline drivers/SmartThings/zigbee-thermostat/src/init.lua:143]: this is actually just a fan mode event, you wouldn't want to hide it

Comment (KKlimczukS) [reply drivers/SmartThings/zigbee-thermostat/src/init.lua:143]: Thanks for catching this!

Comment (KKlimczukS) [reply drivers/SmartThings/zigbee-thermostat/src/init.lua]: Another commit is coming

**Review by greens — APPROVED** [review]

**Comment by greens** [comment]
Don't forget to squash

**Pull request merged**